model/patch_lifecycle.go

```diff
@@ -41,14 +52,20 @@ func VariantTasksToTVPairs(in []patch.VariantTasks) []TVPair {
 // TVPairsToVariantTasks takes a list of TVPairs (task/variant pairs), groups the tasks
 // for the same variant together under a single list, and return all the variant groups
 // as a set of patch.VariantTasks.
-func TVPairsToVariantTasks(in []TVPair) []patch.VariantTasks {
+func (tvp TaskVariantPairs) TVPairsToVariantTasks() []patch.VariantTasks {
```

Comment:
does this need to be a pointer receiver?

Reply:
changed to pointer receiver